.github/workflows/zip.yml

```diff
@@ -0,0 +1,45 @@
+name: ZIP
+
+on:
+  push:
+    tags:
+      - "*"
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install PHP dependencies, optimized
+        run: composer install --no-dev -o
+
+      - name: Install JS dependencies
+        run: yarn
+
+      - name: Build assets for release
+        run: yarn build
+
+      # MUST set in package.json to same as tag name before pushing tag
+      - name: Set version number 
+        run: yarn version:number
+
+      - name: Create build
+        run: yarn package
+
+      - name: Create ZIP
+        uses: papeloto/action-zip@v1
+        with:
+          files: assets/ cf2/ classes/ fields/ languages/ vendor/ processors/ sendwp/ ui/ caldera-core.php readme.txt LICENSE
```

Comment:
@New0 Hold on, if we run yarn package we have everything we need at build/version-number. Wouldn't it be simpler to ZIP that dir?

Reply:
@Shelob9  That would replicate the exact way we're building for production locally

Reply:
Right, that seams better.